src/repositories/community.repository.ts

```
@@ -0,0 +1,13 @@
+import { Model } from 'mongoose';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)